cmake/Modules/opm-material-prereqs.cmake

```diff
@@ -16,8 +16,8 @@ set (opm-material_DEPS
 	# compile with C++0x/11 support if available
 	"CXX11Features REQUIRED"
 	# prerequisite OPM modules
-	"opm-common"
 	"opm-parser"
+	"opm-common"
```

Comment:
> The the order seems wrong to me. 

I agree; initially we had the opposite order of these two libraries and that gave the link line:
```bash
g++ ..... -lopmcommon -lopmparser ...
```

this failed to resolve symbols from opm-common which are required by opm-parser (i.e. the `OpmLog::addMessage()`, something which was fixed by switching the order - but yes; it felt slightly awkward. Maybe this is also related to the `REQUIRED` attribute?